src/select.tsx

```diff
@@ -66,7 +68,7 @@ const useStyles = createUseStyles((theme) => ({
         position: 'absolute',
         top: ({optionsComputedProps}) => optionsComputedProps.top,
         left: ({optionsComputedProps}) => optionsComputedProps.left,
-        width: ({optionsComputedProps}) => optionsComputedProps.width,
```

Comment:
should we limit the label width in this cases? I think is not very important, but...
![image](https://user-images.githubusercontent.com/6722153/132715147-56f106c6-3e74-4277-b218-8080cf90b6dc.png)

Reply:
I think I prefer no limiting it, because it looks better when using the select component:
![Screenshot_20210909_172636](https://user-images.githubusercontent.com/1849576/132715479-09501009-a5fd-4e2c-88a8-10ee84f55cff.png)

Reply:
agree

Reply:
I think the ticket also refers to this problem 

![image](https://user-images.githubusercontent.com/681159/132853042-04b167be-440a-407b-a90e-443a24baa2c1.png)

which may come from a missing `word-space: nowrap`.

Reply:
fixed ;)